src/pages/wishListView/wishListComponent/wishlistComponent.js

```diff
@@ -9,14 +9,19 @@ import {
   Typography,
 } from "@material-ui/core";
 
+import { useRef } from "react";
 import ButtonComponent from "../../../components/Button/ButtonComponent";
 import { DeleteOutline } from "@material-ui/icons";
 import { useHistory } from "react-router";
+// import "./wishListComponent.css";
 const useStyles = makeStyles((theme) => ({
   root: {
     color: "#fff",
     margin: 0,
     borderRadius: "8px",
+    "&.removedItem": {
+      animation: `$removed-item-animation 0.4s cubic-bezier(.55,-0.04,.91,.94) forwards`,
+    },
```

Comment:
Hey, @RohankumarM  animation looks Great but I would suggest fading animation would be the right choice here what you think?

Reply:
I mean everyone has different opinion. Fading would look good too. Let me update my code accordingly and you say what type of animation do you like?

Reply:
> I mean everyone has different opinion. Fading would look good too. Let me update my code accordingly and you say what type of animation do you like?

Hey, @RohankumarM I might not the right person to give suggestions on animation ...........😊 but just have curious about how trying different changes would help us to make the application better so refer to this [link](https://cssanimation.rocks/list-items/) and check the behavior of `Slide down & Fade` to see how the previous index is transitioning to the above index smoothly !!!